README.md

```diff
@@ -18,6 +18,7 @@
       - [USDC on Ethereum](#usdc-on-ethereum)
       - [BTC LNURL](#btc-lnurl)
       - [BTC Address](#btc-address)
+      - [BTC Silent Payment Address](#btc-silent-payment-address-bip-0352)
```

Comment:
@aparkersquare agreed with what you mentioned in #12 about "Silent Payment Address" being more confusing and Static Payment Address making more sense. just flagging here in case we want to call it Static Payment Address. Cool with whatever you decide!

Reply:
The BIP title is "Silent Payments", so I think we should use that here.

If they change the name of the BIP we can definitely update though 😁